app/controllers/verify/finished_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Verify
+  class FinishedController < ApplicationController
```

Comment:
I am not in love with the name of this controller. I think "Finished" is a bit of a misnomer since they aren't actually finished until they enter their USPS verification code. Any ideas for something better are welcome.